pygmt/helpers/utils.py

```diff
@@ -10,6 +10,9 @@
 from collections.abc import Iterable
 from contextlib import contextmanager
 
+import geopandas as gpd
```

Comment:
This `import geopandas` line shouldn't be here at the top-level. I'd suggest importing geopandas in the `return_table` function itself if you need it, and only under `elif data_format=="geopandas"`.
```suggestion
```